src/main/java/com/bookstore/model/Book.java

```diff
@@ -30,4 +32,6 @@ public class Book {
     private String description;
     @Column
     private String coverImage;
+    @Column(name = "is_deleted", nullable = false)
```

Comment:
remove `@Column` above `description` and `coverImage`
```suggestion
    @Column(nullable = false)
```